src/autoyast-rnc/bootloader.rnc

```diff
@@ -2,28 +2,36 @@ default namespace = "http://www.suse.com/1.0/yast2ns"
 namespace a = "http://relaxng.org/ns/compatibility/annotations/1.0"
 namespace config = "http://www.suse.com/1.0/configns"
 
+STRING_BOOL = STRING_ATTR, ("true" | "false")
```

Comment:
we should still
> include "common.rnc"


even if we don't run the validation yet

Reply:
how it will behave in final build which include everything? will be common.rng included many times?